docs/docs/07-forms.md

```diff
@@ -35,7 +35,7 @@ Like all DOM events, the `onChange` prop is supported on all native components a
 
 ## Controlled Components
 
-An `<input>` with `value` set is a *controlled* component. In a controlled `<input>`, the value of the rendered element will always reflect the `value` prop. For example:
+An `<input>` with a `value` property is a *controlled* component:
```

Comment:
Can we leave this as "prop" (makes it clear we're talking about props and not arbitrary properties)